src/main/java/io/quarkus/bot/CheckPullRequestEditorialRules.java

```diff
@@ -132,4 +135,28 @@ private static boolean isUpperCaseException(String title) {
 
         return false;
     }
+
+    private static boolean isMeaningfulPullRequest(GHPullRequest pullRequest) throws IOException {
+        // Note: these rules will have to be adjusted depending on how it goes
+        // we don't want to annoy people fixing a typo or require a description for a one liner explained in the title
+
+        // if we have more than one commit, then it's meaningful
+        if (pullRequest.getCommits() > 1) {
+            return true;
+        }
+
+        // for one liner/two liners, let's be a little more lenient
+        if (pullRequest.getAdditions() <= 2 && pullRequest.getDeletions() <= 2) {
```

Comment:
Component version updates will go through this without notice. We had SmallRye Config updates like that and they were minor version bumps ...

Reply:
In general I agree that we shouldn't put barrier for contributions.

But maybe we should be a bit stricter for changes in https://github.com/quarkusio/quarkus/blob/main/bom/application/pom.xml.
I would mark any PR  that touches `bom/application/pom.xml` as meaningful pull request.

Reply:
I added a commit with that.

Ideally, I think we should deploy and gather feedback and improve from there.

Reply:
@rsvoboda I let you check the last commit does what you wanted and then I would be in favor of merging and deploying.